build.gradle

```diff
@@ -14,6 +14,7 @@ plugins {
   id 'org.sonarqube' version '5.0.0.4638'
   id 'project-report'
   id 'io.codearte.nexus-staging' version '0.30.0'
+  id 'org.cyclonedx.bom' version '1.7.2'
```

Comment:
why not latest (1.8.2)?

Reply:
Updated. I copied from an example somewhere which seems to be a bit older.